Package@swift-5.6.swift

```diff
@@ -0,0 +1,15 @@
+// swift-tools-version:5.6
+import PackageDescription
+
+let package = Package(
+  name: "Lottie",
+  platforms: [.iOS("11.0"), .macOS("10.10"), .tvOS("11.0")],
+  products: [
+    .library(name: "Lottie", targets: ["Lottie"]),
+  ],
+  dependencies: [
+    .package(url: "https://github.com/airbnb/swift", branch: "master"),
```

Comment:
We probably either need to check in the `Package.resolved` file, or reference a specific commit here. Otherwise builds on Lottie master could start failing because we updated the Style repo. If I can't get the `Package.resolved` to work properly I'll update this to reference a specific commit.